faculty/clients/job.py

```diff
@@ -398,6 +398,22 @@ def get_subrun(
         )
         return self._get(endpoint, _SubrunSchema())
 
+    def cancel_run(self, project_id, job_id, run_identifier):
+        """Cancel a running job.
+
+        Parameters
+        ----------
+        project_id : uuid.UUID
```

Comment:
With the merging of the docs PR, we now include descriptions for all parameters. Can you copy the project_id and job_id descriptions from other methods into this docstring?

Reply:
Sure thing, sorry, I should have looked at the other changes more closely!